templates/robotics/index.html

```diff
@@ -14,728 +16,695 @@
+          </div>
+          <p>
+            Ubuntu is an open-source operating system. For more than a decade, Ubuntu has supported key robotics open-source projects, such as <a href="/robotics/what-is-ros">ROS</a>, PX4, Autoware, Open-CV, PCL, and more. Every aspect of the development process benefits from Ubuntu's responsiveness, ease of use, regular software updates, lightweight nature, and a high degree of security.
+          </p>
+          <div class="p-cta-block">
+            <a href="/robotics/docs" class="p-button--positive">Explore documentation</a> <a href="/desktop">Learn more about Ubuntu&nbsp;&rsaquo;</a>
           </div>
         </div>
-        <ul class="p-list">
-          <li class="p-list__item">
-            <a href="/engage/drone-infrastructure-case-study-smartdrone">
-              SmartDrone fast-tracks drone infrastructure revolution with Ubuntu Core
-            </a>
-          </li>
-          <li class="p-list__item">
-            <a href="/engage/dockerandros">Docker &amp; ROS: When all you have is a hammer, everything looks like a nail</a>
-          </li>
-        </ul>
       </div>
-      <div class="col-4 p-divider__block">
-        <div class="p-heading-icon--muted">
-          <div class="p-heading-icon__header">
-            {{ image(url="https://assets.ubuntu.com/v1/6e184942-Webinar.svg",
+      <div id="robotics-snapcraft"
+           class="p-tabs__content"
+           role="tabpanel"
+           aria-labelledby="robotics-tab-snapcraft">
+        <div class="p-strip is-shallow">
+          <div class="u-hide--small u-hide--medium">
+            {{ image(url="https://assets.ubuntu.com/v1/96a4e5bc-diagram-snapcraft.png",
                         alt="",
-                        width="32",
-                        height="28",
+                        width="1800",
+                        height="1200",
                         hi_def=True,
-                        attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
-                        loading="lazy",) | safe
+                        loading="lazy") | safe
             }}
-            <h3 class="p-heading-icon__title p-heading--4">Video and Webinar</h3>
+          </div>
+          <p>
+            Snapcraft is the command-line tool for packaging your software as a Snap. Snaps are containerised software packages designed for embedded applications, with increased performance compared to <a href="/engage/dockerandros">other technologies</a>. Snaps give you all the interfaces you need to bundle your dependencies in one package, while providing secure interfaces to access the host disc and privileged resources.
+          </p>
+          <div class="p-cta-block">
+            <a href="https://canonical-robotics.readthedocs-hosted.com/en/latest/how-to-guides/"
+               class="p-button--positive">Explore documentation</a> <a href="https://canonical-robotics.readthedocs-hosted.com/">Learn more about Snapcraft&nbsp;&rsaquo;</a>
```

Comment:
For readability purposes, we should split up the anchor tags on separate lines
```suggestion
               class="p-button--positive">Explore documentation</a> 
               <a href="https://canonical-robotics.readthedocs-hosted.com/">Learn more about Snapcraft&nbsp;&rsaquo;</a>
```